src/Model/Lotto.js

```diff
@@ -0,0 +1,56 @@
+const LOTTO = require('../constant/constant');
+
+class Lotto {
+  #numbers;
+
+  constructor(numbers) {
+    this.validate(numbers);
+    this.#numbers = numbers;
+  }
+
+  get lottoNumbers() {
+    return this.#numbers;
+  }
+
+  validate(numbers) {
+    this.checkLength(numbers);
+    this.checkIsNumber(numbers);
+    this.checkDuplicate(numbers);
+    this.checkNumberRange(numbers);
+  }
+
+  checkLength(numbers) {
+    if (numbers.length !== LOTTO.LENGTH) {
+      throw new Error('[ERROR] 로또 번호는 6개여야 합니다.');
+    }
+  }
+
+  checkDuplicate(numbers) {
+    const set = new Set();
+    numbers.forEach((number) => {
+      set.add(number);
+    });
+
+    if (set.size != LOTTO.LENGTH) {
+      throw new Error('[ERROR] 로또 번호는 중복되면 안됩니다. ');
+    }
+  }
+
+  checkIsNumber(numbers) {
+    const checker = /^[0-9]+$/;
+    const checkedArray = numbers.filter((n) => checker.test(n));
+    if (checkedArray.length !== LOTTO.LENGTH)
+      throw new Error('[ERROR] 로또 번호는 숫자여야 합니다.');
+  }
```

Comment:
단순히 numbers 배열 내의 모든 숫자들이 숫자인지 확인하는 로직이니까 Array.every()를 사용해보시는 것도 좋을 것 같아요!

```suggestion
  checkIsNumber(numbers) {
    const checker = /^[0-9]+$/;
    const isValidNumbers = numbers.every((number) => checker.test(number));
    
    if (!isValidNumbers) {
      throw new Error('[ERROR] 로또 번호는 숫자여야 합니다.');
    }
```

Reply:
오every라는 함수가 있는것을 몰랐네요 좋은 인사이트 감사해요!